ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -527,6 +527,42 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-ipsec
+- as: gcp-ipi-user-labels-tags-tp-amd-f28-destructive
```

Comment:
can we add "filestore-csi" keyword in the ci job name, so that we easily identify the enabled enabled.